tests/test_reactive_static_adi.R

```diff
@@ -0,0 +1,40 @@
+library(dplyr)
+library(testthat)
+
+source(here::here("src/data_processing.R"))
+source(here::here("src/auto_scaling_algorithm.R"))
+source(here::here("src/calculate_adi.R"))
+source(here::here("tests/setup_test.R"))
+
+# Load data files
+data <- processing_data("data/util_data.csv")
+decreasing_data <- read.csv("data/decreasing_data.csv")
```

Comment:
as a test, fine. But, it would be better to have the data used in the test below the tests directory (It is ok to have a copy)

Reply:
Done. Copied data files to the tests directory